packages/react/README.md

```diff
@@ -128,21 +128,18 @@ function MyComponent() {
    const name: string = useFlag('user.name', '');
    ```
 
-### Server-Side Rendering Support (experimental)
```

Comment:
I think we can keep the "experimental" state as we're recommending the OpenFeature integration.